test/helpers/filesystem.go

```diff
@@ -0,0 +1,25 @@
+package helpers
+
+import (
+	"github.com/spf13/afero"
+)
+
+const (
+	DummyMachineID = "dummy-machine-id"
+	DummyAgentID   = "779cdd70-e9e2-58ca-b18a-bf3eb3f71244"
+
+	machineIDPath = "/etc/machine-id"
```

Comment:
This declaration is being duplicated. Would it make sense to make it public from `agent` package and use it here?

Reply:
I wouldn't. This is an internal concept. Better a small duplication here in my opinion

Reply:
ok